tests/test-update-install-flatpaks.c

```diff
@@ -2189,6 +2189,254 @@ test_update_deploy_flatpaks_on_reboot (EosUpdaterFixture *fixture,
   g_assert (g_strv_contains ((const gchar * const *) deployed_flatpaks, flatpaks_to_install[0].app_id));
 }
 
+#define EUFI_EXIT_CODE_APPLY_FAILED 4
```

Comment:
Nitpick: I’d add a comment above this like `/* See ``man 8 eos-updater-flatpak-installer``. */`

Reply:
(Fixed locally.)